ansible/register_flavors.yml

```diff
@@ -0,0 +1,11 @@
+---
+- hosts: localhost
+  tasks:
+    - name: Register Nova flavors
+      include_role:
+        name: nova-flavors
```

Comment:
I did find stackhpc.os-flavors but it seemed to only really added a wrapper around the ansible_python_interpreter setting/unsetting, which we don't need here anyway, and I'd still need to do the extra_specs fiddling, so I didn't think it would add much benefit.